ambry-cloud/src/main/java/com.github.ambry.cloud/VcrServer.java

```diff
@@ -146,11 +146,13 @@ public void startup() throws InstantiationException {
 
       StoreKeyConverterFactory storeKeyConverterFactory =
           Utils.getObj(serverConfig.serverStoreKeyConverterFactory, properties, registry);
-
+      CloudBlobCryptoAgentFactory cloudBlobCryptoAgentFactory =
+          Utils.getObj(cloudConfig.cloudBlobCryptoAgentFactoryClass, properties, clusterMapConfig.clusterMapClusterName,
+              clusterMap.getMetricRegistry());
       cloudBackupManager =
-          new CloudBackupManager(cloudConfig, replicationConfig, clusterMapConfig, storeConfig, storeKeyFactory,
-              clusterMap, virtualReplicatorCluster, cloudDestinationFactory, scheduler, connectionPool, registry,
-              notificationSystem, storeKeyConverterFactory, serverConfig.serverMessageTransformer);
+          new CloudBackupManager(properties, cloudConfig, replicationConfig, clusterMapConfig, storeConfig,
+              storeKeyFactory, clusterMap, virtualReplicatorCluster, cloudDestinationFactory, scheduler, connectionPool,
+              registry, notificationSystem, storeKeyConverterFactory, serverConfig.serverMessageTransformer, metrics);
```

Comment:
Looks like metrics is not necessary here. Only  metrics.vcrStartTimeInMs used in this function, but it's a one time display. 

How about just remove vcrMetrics from this function? CloudBlobStore can create vcrMetrics based on given registry.

Reply:
Removed the two metrics.  Also creating metrics object in cloud backup manager to share across CloudBlobStores.